client/app/telecom/telephony/alias/configuration/changeType/telecom-telephony-alias-configuration-changeType.html

```diff
@@ -1,104 +1,127 @@
-<section class="telephony-group-alias-configuration-change-type">
+<div class="text-center"
+     data-ng-if="$ctrl.loading">
+    <oui-spinner></oui-spinner>
+</div>
 
-    <header>
-        <section-back-link section-back-link-to-state="telecom.telephony.alias.configuration"
-                           section-back-link-title="{{ 'telephony_alias_configuration_back_link' | translate }}">
-        </section-back-link>
-        <h1 data-translate="telephony_alias_change_type_title"></h1>
+<div class="container-fluid ml-0"
+     data-ng-if="!$ctrl.loading">
+    <toast-message></toast-message>
 
-    </header>
-
-    <div class="text-center" data-ng-if="AliasConfigurationChangeTypeCtrl.loading.line">
-        <oui-spinner></oui-spinner>
+    <div class="row">
+        <h2 class="col-lg-5 col-sm-12 oui-header_2"
+            data-translate="telephony_alias_change_type_title"></h2>
     </div>
 
-    <div class="telecom-section-content"
-         data-ng-if="!AliasConfigurationChangeTypeCtrl.loading.line">
+    <div class="row">
+        <oui-collapsible class="col-lg-8 col-sm-12 ml-0 mr-2 mb-4"
+                         data-expanded="false"
+                         data-heading="{{:: 'telephony_alias_change_type_subtitle' | translate }}">
+            <p class="oui-paragraph"
+               data-translate="telephony_alias_change_type_details"></p>
+        </oui-collapsible>
+    </div>
 
-        <toast-message></toast-message>
+    <div class="form-group">
+        <oui-button class="ml-0"
+                    data-on-click="$ctrl.switchOptionMode()"
+                    data-variant="secondary">
+            <span data-translate="{{ ($ctrl.showExpertOptions ? 'telephony_alias_change_type_show_simple_option' : 'telephony_alias_change_type_show_expert_option') | translate }}"></span>
+        </oui-button>
+    </div>
 
-        <div class="alert alert-info"
-              role="alert">
-            <p data-translate="telephony_alias_change_type_info_text_1"></p>
-            <p data-translate="telephony_alias_change_type_info_text_2"></p>
+    <div data-ng-if="!$ctrl.showExpertOptions">
+        <div class="row w-100 d-md-flex">
+            <div class="col-lg-4 col-sm-12 mb-4">
```

Comment:
useless `col-sm-12`.

Reply:
No the two of them are needed, here it is a screen shot of html rendering if i delete one of the classes => 
without `col-sm-12` : https://dl.plik.ovh/file/P5M3rfxkaZrRebxh/Lr60FY6MNyCLgQN1/Without%20col%20sm%2012.png